sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## 5.1.0-beta.1 (Unreleased)
 ### Breaking changes
-- It defaults to the latest supported API version, which currently is `3.1-preview.1`.
+- It defaults to the latest supported API version, which currently is `3.1-preview.2`.
```

Comment:
nit: Do we want to consider upgrading to a newer version as a new feature change?

Reply:
Newer version of what? Not sure I follow

Reply:
Moved to `New Features`